server/src/main/java/org/jboss/as/server/moduleservice/ServiceModuleLoader.java

```diff
@@ -130,10 +130,10 @@ protected Module preloadModule(final String name) throws ModuleLoadException {
 
     @SuppressWarnings("unchecked")
     @Override
-    public ModuleSpec findModule(ModuleIdentifier identifier) throws ModuleLoadException {
-        ServiceController<ModuleDefinition> controller = (ServiceController<ModuleDefinition>) serviceContainer.getService(moduleSpecServiceName(identifier));
+    public ModuleSpec findModule(String name) throws ModuleLoadException {
+        ServiceController<ModuleDefinition> controller = (ServiceController<ModuleDefinition>) serviceContainer.getService(moduleSpecServiceName(name));
```

Comment:
I think this method should have been deprecated for removal calling internally to the one using the String as an argument. Otherwise, any caller currently using the old one will now be using the one provided by the parent class.